deploy/deploy-api.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Deploy API Server on GCP.
+
+if (( $# != 1 ))
+then
+  echo "Usage: deploy/deploy-api.sh <dataset>"
```

Comment:
Add a `./`

Reply:
Not needed. In the past when I requested this, it was needed